meta-mel-support/recipes-core/images/mel-image.inc

```diff
@@ -11,4 +11,4 @@ inherit core-image image-sanity-incompatible
 
 IMAGE_FEATURES .= "${@bb.utils.contains('COMBINED_FEATURES', 'alsa', ' tools-audio', '', d)}"
 IMAGE_INSTALL += " util-linux-mkfs connman ${@bb.utils.contains("INCOMPATIBLE_LICENSE", "GPLv3", "", "haveged", d)}"
-IMAGE_FEATURES += "splash"
+IMAGE_FEATURES += "${@bb.utils.contains('EXTRA_IMAGE_FEATURES', 'graphics', 'splash', '', d)}"
```

Comment:
This should probably go based on IMAGE_FEATURES. Directly going based on EXTRA_IMAGE_FEATURES limits options, i.e. we couldn't create an image that's always graphical. I don't intend to do that, though, so this is fine for now, but for future reference.

Reply:
@kergoth  core_image-base does not follow this. Do you think we should introduce it. I was thinking not to merge this.

Reply:
Agreed, that's fine.